.circleci/docker/common/install_cuda.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
```

Comment:
Oh is this just copied over from `pytorch/builder`?

Might be useful to prune the versions we don't need from this script to make it more manageable

Reply:
Yes, maybe we can call builder script instead ? If we add this file here, every time we update the version we would need to update it in both place

Reply:
Yes, it is copied over from `pytorch/builder`. I will clean up the other unused versions.

Reply:
Is it possible to keep this in pytorch/builder (don't copy) but call git clone of builder repo and call this script from you docker install ? In most of our workflows we clone builder and pytorch. So you normally have builder available

Reply:
That makes sense. Let me try to re-use the `install_cuda.sh` from builder repo, instead of copying it over here.

Reply:
That makes sense. Let me try to re-use the install_cuda.sh from builder repo, instead of copying it over here.

Reply:
> Is it possible to keep this in pytorch/builder (don't copy) but call git clone of builder repo and call this script from you docker install ? In most of our workflows we clone builder and pytorch. So you normally have builder available

I think that would be a bit too much to do a full git clone. A curl of the raw file should probably suffice for now until we can fully migrate it over